internal/handlers.go

```diff
@@ -102,6 +105,183 @@ func PostAccount(w http.ResponseWriter, r *http.Request, params httprouter.Param
 		return
 	}
 
-	Accounts[account.AccountId] = account
+	models.Accounts[account.AccountId] = account
 	utils.WithPayload(w, http.StatusCreated, nil)
 }
+
+func GetAllTransactions(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	transactions := []*models.Transaction{}
+
+	accountId := params.ByName("account-id")
```

Comment:
*minor* : I like to use constant for any string or magical number that we use because
-  it helps to keep all those strings that we could use in multiple places (including tests), in a single place, so we can update them if necessary
- Those strings will be kept through the life cycle of your system, so the garbage collector doesn't need to keep creating a new space in memory and deleting it for each request. So it can be more memory-efficient. The impact is probably minimal, but it will grow for more request your service receives

Reply:
accountID